src/material.cc

```diff
@@ -12,18 +12,20 @@ namespace cyclus {
 
 const ResourceType Material::kType = "Material";
 
+static int default_package_id_ = 1;
```

Comment:
This can only exist in once place, so this may not be the best place.  There are other examples in the code base.

It may also make sense to make it a static member variable of the `Package` class (which could mean that the one place it exists is the `package.cc` file.